TextToSpeech/TextToSpeechJsonRpc.cpp

```diff
@@ -204,9 +235,18 @@ namespace Plugin {
 
         if(parameters.HasLabel("primvolduckpercent")) {
            int8_t  primVolDuck;
-           primVolDuck = (int8_t) std::stoi(GET_STR(parameters,"primvolduckpercent","-1"));
-           _tts->SetPrimaryVolDuck(primVolDuck);
+           InputValidation::Instance().addValidator("primvolduckpercent", ExpectedValues<std::string>("^-?[0-9]+$"));
```

Comment:
Could we keep all the addValidator in TextToSpeechImplementation::Configure() API?